vignettes/introduction-to-partition.Rmd

```diff
@@ -69,56 +71,90 @@ df <- simulate_block_data(
   upper_corr = .6,
   n = 100
 )
+```
+
+In a heatmap showing the correlations between the simulated variables, blocks of correlated features are visible:
+
+```{r}
+ggcorrplot::ggcorrplot(corr(df))
+```
+
+Many types of data follow a pattern like this. Closely related to the block correlation structure found in genetic data is that found in microbiome data. The data set `baxter_otu` has microbiome data on 172 healthy patients. Each row represents a patient, and each column represents an Operational Taxonomic Unit (OTU). OTUs are species-like relationships between bacteria determined by analyzing their RNA. Each cell in the data represents the logged-count of the OTU found in the patient's stool sample, and there are 1,234 OTUs in all.
```

Comment:
Suggestion:
Each cell in the dataset represents the logged-count of an OTU found in a patient's stool sample with 1,234 OTUs in all.